modules/services/fusuma.nix

```
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+let cfg = config.services.fusuma;
+in with lib; {
+  meta.maintainers = with maintainers; [ iosmanthus ];
+  options.services.fusuma = {
+    enable = mkEnableOption
+      "the fusuma systemd service to automatically enable touchpad gesture";
+    package = mkOption {
```

Separate option definition by empty lines:
```suggestion

    package = mkOption {
```